models/config.yaml

```diff
@@ -0,0 +1,25 @@
+.*:
+  wbits: 'None'
+  model_type: 'None'
+  groupsize: 'None'
+  pre_layer: 0
+  mode: 'cai-chat'
+.*-4bit-128g:
+  wbits: 4
+  groupsize: 128
+.*-128g-4bit:
```

Comment:
I'd swap to `(gr128|128g)` to account for both ways it's commonly written.

Also, `3bit` gr128 models also exist and are picking up popularity (Notably, llama-30b-3bit-gr128 can fit max context length on a 3090/4090, whereas 4bit hits VRAM limit at just over 1000 token context, so 3bit is preferable for that)

Reply:
Done here https://github.com/oobabooga/text-generation-webui/pull/1167/commits/2f27df702a1c765586b298422db8a0118fafa643